tests/bootstrap.php

```diff
@@ -0,0 +1,14 @@
+<?php
```

Comment:
It's not necessary. You can just add one line to config
https://github.com/yiisoft/http/blob/master/phpunit.xml.dist#L6